EIPS/eip-list-supported-methods.md

```diff
@@ -0,0 +1,99 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7663
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.